specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/syntex.json

```diff
@@ -0,0 +1,493 @@
+              "$ref": "#/definitions/DocumentProcessorResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DocumentProcessorResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DocumentProcessor_Create": {
+            "$ref": "./examples/DocumentProcessor_Create.json"
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates a document processor resource for a given name.",
+        "operationId": "DocumentProcessor_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ProcessorNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "description": "Document processor details.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DocumentProcessorPatchableProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DocumentProcessorResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DocumentProcessor_Update": {
+            "$ref": "./examples/DocumentProcessor_Update.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Syntex/operations": {
+      "get": {
+        "operationId": "Operations_List",
+        "description": "List of all REST operations supported by the Microsoft.Syntex provider.",
+        "tags": [
+          "Operations"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Describes the result of a successful operation.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
>ErrorResponse [](http://example.com/codeflow?start=37&length=13)

Can potentially refer directly to "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"

Reply:
Thank you, referencing ErrorResponse from common-types v3 worked!